pandas/tests/generic/test_generic.py

```diff
@@ -1019,3 +1019,17 @@ def test_pipe_panel(self):
 
             with pytest.raises(ValueError):
                 result = wp.pipe((f, 'y'), x=1, y=1)
+
+    @pytest.mark.filterwarnings("ignore:\\nPanel:FutureWarning")
```

Comment:
this is ok, but pls parameterize over box (e.g. pd.Series, pd.DataFrame). ok to not test on Panel.